src/Features/CSharp/Portable/LanguageServices/CSharpSymbolDisplayService.SymbolDescriptionBuilder.cs

```diff
@@ -109,7 +109,7 @@ protected override void AddDescriptionForProperty(IPropertySymbol symbol)
                 else
                 {
                     var fullParts = ToMinimalDisplayParts(symbol, s_propertySignatureDisplayFormat);
-                    var neededParts = fullParts.SkipWhile(p => p.Symbol == null);
```

Comment:
I see.  This code is very unfortunate.  It's attempting to change our QuickInfo so that we show accessibility for accessors, while still keeping it off for the property itself.  So hacky.  I'm working on a better way.